<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumWidget</class>
 <widget class="QWidget" name="AlbumWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>40</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="iconLabel">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
